File: client/common/model/sku/sku.draft.ts

export class SkuDraft {
    id: number|undefined;
    resourceId: number|undefined;
    skuItemId: string|undefined;
    skuDraftId: string|undefined;
    status: string|undefined;
    
    constructor(id: number|undefined, resourceId: number|undefined, skuItemId: string|undefined, skuDraftId: string|undefined, status: string|undefined) {
        this.id = id;
        this.resourceId = resourceId;
        this.skuItemId = skuItemId;
        this.skuDraftId = skuDraftId;
        this.status = status;
    }
}

File: server/internal/sku/models/sku_draft.go
package models

import "server/common/middleware/database"

type SkuDraft struct {
	database.BaseEntity
	SkuItemId  string `json:"sku_item_id"`
	SkuDraftId string `json:"sku_draft_id"`
	Status     string `json:"status"`
	ResourceId int64  `json:"resource_id"`
}

func (s *SkuDraft) TableName() string {
	return "sku_draft"
}
